### PR TITLE
reclaim the flatpak

### PR DESCRIPTION
**Pull request by dada513**
secured the metaserver

### CODE REVIEW

File: flathub.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "only-arches": ["x86_64"],
-  "publish-delay-hours": 1
+  "end-of-life": "PolyMC is deprecated due to a hostile takeover by the lead dev; a replacement will soon be released. More info at https://discord.gg/kVGPVjWN3a"
```
[the remark this answers is not in the document]
Comment (TayouVR) [reply flathub.json:3]: ofc this would have to wait for the PrismLauncher package actually being made.

**Comment by orowith2os** [comment]
A release has been made, looks fine to use that and start making a new submission to flathub: https://github.com/PrismLauncher/PrismLauncher/releases/tag/5.0

**Comment by orowith2os** [comment]
Turns out @Scrumplex is already on it: https://github.com/flathub/flathub/pull/3596

**Comment by binex-dsk** [comment]
Closing as this points to a meta server that isn't a part of PolyMC.

**Comment by dada513** [comment]
You are not a maintainer of the flatpak. You should not have access.

**Comment by HeyaGlitz** [comment]
> You are not a maintainer of the flatpak. You should not have access.

He is an official maintainer of PolyMC and is in the PolyMC organization. I think he has all the rights to control the outcome of this.

**Comment by KaspianDev** [comment]
Uh? Why are you modifying flatpak instead of adding new application?

Polymc still exists and there's no point of "reclaiming" it. I'd let actual members of the project or ideally owner decide.

**Comment by dada513** [comment]
Plan was to deprecate it and resubmit after changing meta server in a final update, so that no malware can be pushed through the meta system 

polymc exists but there has been a hostile takeover of the project

**Comment by basdxz** [comment]
> Plan was to deprecate it and resubmit after changing meta server in a final update, so that no malware can be pushed through the meta system
> 
> polymc exists but there has been a hostile takeover of the project

PolyMC has yet to have any code changes pushed since the incident, and the meta server has not been updated to my knowledge. What gives it less trust compared to the less than a week old fork?

**Comment by KaspianDev** [comment]
> Plan was to deprecate it and resubmit after changing meta server in a final update, so that no malware can be pushed through the meta system
> 
> polymc exists but there has been a hostile takeover of the project

1. The project is not abandoned, and someone else wants to maintain this flatpak
2. Any project can have malware at any time, no matter what happens, I don't know why talk about something that did not happen and most likely will not happen.

**Comment by CytexalMika** [comment]
>     1. The project is not abandoned, and someone else wants to maintain this flatpak
This flatpak is de-facto abandoned and the current maintainers (maybe) don't trust new applicant.
Decommission sounds like the logical solution to me.
Upstream can just resubmit PolyMC to flathub if they really want to after that.

**Comment by KaspianDev** [comment]
> > ```
> > 1. The project is not abandoned, and someone else wants to maintain this flatpak
> > ```
> 
> This flatpak is de-facto abandoned and the current maintainers (maybe) don't trust new applicant. Decommission sounds like the logical solution to me. Upstream can just resubmit PolyMC to flathub if they really want to after that.

Maybe, but definetly not rebranding this repo to prism.

**Comment by orowith2os** [comment]
> > > ```
> > > 1. The project is not abandoned, and someone else wants to maintain this flatpak
> > > ```
> > 
> > 
> > This flatpak is de-facto abandoned and the current maintainers (maybe) don't trust new applicant. Decommission sounds like the logical solution to me. Upstream can just resubmit PolyMC to flathub if they really want to after that.
> 
> Maybe, but definetly not rebranding this repo to prism.

Better to give the end users a working app while flatpak tells them to switch to something else while it's being EOLed. The maintainers at the upstream PolyMC project are *not* fit to maintain this, nor any distribution-specific package, especially considering there have since been PRs to the PolyMC repo that remove DRM checks and the controller of the repo (even if jokingly) said "LGTM". 

And not having malicious code pushed doesn't make it "fine". If there's only one person that has full control over the repository, with no checks needing to be done, who says that there can't be malicious code in the future? Who checked to make sure that the commit removing the code of conduct was safe? Who made sure the MSA ID was safe in the latest commit? Who can do any of those in the future, and revert them, without being shot down my the upstream maintainers?

**Comment by binex-dsk** [comment]
> You are not a maintainer of the flatpak. You should not have access.

I've been the maintainer since the beginning, and I wish to continue maintenance of this project. You have every right to go to maintaining PrismLauncher's Flatpak if you wish.

**Pull request closed without merging**